content/SCALE/SystemSettings/ScaleShell.md

```diff
@@ -4,42 +4,45 @@ geekdocCollapseSection: true
 weight: 60
 ---
 
-The SCALE shell is a convenient means to run command lines tools, configure different system settings, or find log files or debug information.
-The shell opens with the root user logged in.
+The SCALE **Shell** is convenient for running command lines tools, configuring different system settings, or finding log files and debug information.
+The **Shell** opens with the root user logged in.
+
+{{< hint danger >}}
+**Warning: the supported mechanisms for making configuration changes are the TrueNAS WebUI, CLI, and API exclusively. ALL OTHERS ARE NOT SUPPORTED AND WILL RESULT IN UNDEFINED BEHAVIOR AND MAY RESULT IN SYSTEM FAILURE.**
```

Comment:
```suggestion improper use of bold and all caps. emphasis is already in the hint tags used.
Warning: the supported mechanisms for making configuration changes are the TrueNAS WebUI, CLI, and API exclusively. *All others are not supported and result in undefined behaviors, and might result in system failure.* 
```

Reply:
@micjohnson777 
I know the text isn't optimized, but that's exactly how it looks in the Console Setup Menu, so we should probably leave that one as-is.